packages/rancher-istio/charts/templates/istio-install-psp.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: istio-installer
```

Comment:
Wondering if we should expose this in `values.yaml`
(and its references throughout the other file(s))

Reply:
Resolving this in favor of discussion: https://github.com/rancher/charts/pull/957#discussion_r597952119